src/python/pants/util/contextutil.py

```diff
@@ -22,6 +22,13 @@
 from pants.util.tarutil import TarFile
 
 
+class InvalidZipPath(zipfile.BadZipfile):
+  """Raise instead of passing unchecked falsey values through to zipfile.ZipFile.
```

Comment:
The doc here reads like a bug report, instead prefer language that says what the exception means. Perhaps:
```python
"""Indicates a bad zip file path."""
```